.zuul.d/dump_zuul_vars.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: controller
+  gather_facts: true
+  tasks:
+    - name: Dump Zuul Vars
+      ansible.builtin.copy:
+        src: "{{ zuul.executor.log_root }}/zuul-info/inventory.yaml"
+        dest: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}/scenarios/centos-9/zuul_inventory.yml"
+
+    - name: Slurp inventory
```

Comment:
I don't really get it - why do we copy, then slurp, then overwrite?

Reply:
zuul-info/inventory.yaml exists on zuul executor. 
We need to copy this file on the job node.
and after that, I am removing these contents from [zuul_inventory](https://logserver.rdoproject.org/36/136/48900bdb6f15d6a4ce661e78639fd172fa421100/github-check/end-to-end/bbe7d90/zuul-info/inventory.yaml)
```
all:
  hosts:
    controller:
      ansible_connection: ssh
      ansible_host: 38.102.83.8
      ansible_port: 22
      ansible_python_interpreter: auto
      ansible_user: zuul-worker
      crc_parameters: --memory 16000 --disk-size 120 --cpus 6
      nodepool:
        az: nova
        cloud: vexxhost-nodepool-tripleo
        external_id: b88c3966-9e1e-4ba1-b33c-e0b7ecb49f9a
        host_id: 7be6eb536a89b1266edff7cfa16e93a8ea0da5df2cfadeeb194a3ffc
        interface_ip: 38.102.83.8
        label: centos-9-stream-crc-xxl
        private_ipv4: 38.102.83.8
        private_ipv6: null
        provider: vexxhost-nodepool-tripleo
        public_ipv4: 38.102.83.8
        public_ipv6: ''
        region: RegionOne
```
and dumping the zuul vars in that file. That's why I have done that.